### ISSUE TITLE
Clarify whether term values should be string literals or URI references

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tdwg/ac#70]
```
1. Provide an ac Term Name or Label. xmpRights:UsageTerms, dcterms:type, 
dcterms:rights, dcterms:source, dcterms:creator, dcterms:language, 
dcterms:temporal, and dcterms:format, and probably others like 
xmpRights:WebStatement

2. Describe the defect or lack of clarity you find in the term. 

Comment by Cyndy Parr <parrc@si.edu> during public comment:

Re: License Terms
EOL uses the same URI to hold a URL pointer to a CC license such as
(see http://eol.org/schema/media_extension.xml). In the AC notes, you
seems to conflate the license itself and the text to display about the
license. Would be best to clarify in the notes -- if both are
acceptable, then show examples of both.

Comment from Steve:
This question is really touching on a broader issue that Audubon Core is 
currently dodging, namely whether the values given for some terms should be 
"non-literal value surrogate value strings" (sensu DCMI Abstract model, see 
http://code.google.com/p/tdwg-rdf/wiki/DublinCore#1.1.4.2._Structure_of_DCAM_non
-literal_value_surrogates ) or URI references to a non-literal resource.  The 
definitions of dcterms:type, dcterms:rights, dcterms:source, dcterms:creator, 
dcterms:language, dcterms:temporal, and dcterms:format are all declared by DCMI 
to have non-literal values, which according to the DCMI Abstract Model (DCAM) 
means that their values should be represented by zero to one value URIs and 
zero or more value strings.  As a practical matter, as far as I know RDF is the 
only representation of the DCAM that anybody uses.  In the RDF representation 
of the DCAM, the object of terms with non-literal values is either a 
URI-reference or a blank node, but NOT a literal (DCAM handles the value 
strings in a somewhat odd manner). 

So this issue to some extent boils down to a problem that will come up when 
Audubon Core tries to create an RDF guide for the standard.  In some of the 
cases of dcterms: terms incorporated into Audubon Core (e.g. dcterms:language), 
there are well-recognized controlled value strings that people will probably 
just use and won't care about URIs.  But in several cases (dcterms:rights, 
dcterms:source, and dcterms:creator) there will be some people who might 
actually want to use URIs rather than strings, or perhaps even both.  So how 
will Audubon Core handle that?  In the case of something like dcterms:creator, 
one could fall back on dc:creator for the string version ("Steven J. Baskauf") 
and use dcterms:creator for the URI (http://viaf.org/viaf/63557389 ).  But not 
all of the dcterms: terms have dc: analogues.  In any case, the AC 
documentation tells people to use strings for dcterms: terms that probably 
should have URI-reference values. 

In the particular case of xmpRights:UsageTerms, the situation is really messy.  
Since the xmpRights: terms don't dereference to RDF, I don't know how to see 
how it is defined in RDF, but the pdf normative document says that its value 
type is not text but rather "Language Alternative" which section 8.2.2.4 shows 
to be a complex RDF beast involving one of the horrid RDF Container classes.  
In contrast, xmpRights:WebStatement, which I would expect to be a URI is 
declared to have a value type of "text" because of "historical usage".  This 
seems to be completely backwards to me.  If I were advising EOL, I would say to 
use xmpRights:UsageTerms for a (possibly xml language coded) string literal 
describing the licensing terms and xmpRights:WebStatement for the URI version 
of the license with that URI as a URI-reference rather than a string literal if 
represented in RDF.  That is practically the opposite of what the normative 
documentation says to do but pretty close to what the AC guidelines suggest. 

Basically, I think that there are two options here for the authors.  Dig in and 
figure out a rigorous solution to the string vs. URI problem now, or leave 
things as ambiguous as they are now and then struggle with it at some future 
point when they try to write an RDF guide.  TDWG does not have any requirement 
that vocabularies have an RDF guide, but I suspect people will be wanting one 
sooner or later.


```

Original issue reported on code.google.com by `steve.ba...@vanderbilt.edu` on 25 Apr 2013 at 4:42
